src/components/App.js

```diff
@@ -78,25 +80,27 @@ const App = () => {
 
   return (
     <div className="app-container">
-      <h1>Ducks 'R' Us</h1>
-      <p>API Status: {APIHealth}</p>
-      <Router>
-      <div className="navabr">
-        <Link to="/login">Login</Link>
-        <Link to="/register">Register</Link>
-        <Link to="/products">Products</Link>
-        {/* <Link to="/cart">My Cart </Link> */}
+      <div>
+        <Navbar />
       </div>
+      <p>API Status: {APIHealth} ***We can take this off whenever.***</p>
+      <Router>
+        <div className="navabr">
+          <Link to="/login">Login</Link>
+          <Link to="/register">Register</Link>
+          <Link to="/products">Products</Link>
+          {/* <Link to="/cart">My Cart </Link> */}
+        </div>
```

Comment:
Since the navbar is in place, these links can be removed

Reply:
Agreed. I was working on the Togglewithit branch earlier and thought the same thing. I updated this in that branch.